package.json

```diff
@@ -39,10 +39,12 @@
     },
     "dependencies": {
         "@improbable-eng/grpc-web": "^0.12.0",
+        "@reduxjs/toolkit": "^1.3.4",
```

Comment:
I'm not super happy about this but it might indeed make things a little easier. Can you also add `redux-saga` per the bug too please?

Reply:
I'm actually quite against `toolkit` now. I don't think we need what it does. Please remove (see https://react-redux.js.org/introduction/basic-tutorial) for a basic redux guide